api/src/lib/apikey.mjs

```diff
@@ -108,17 +108,19 @@ export async function deleteApikey(id = false) {
  * @param {object} data - The data to save for the API key
  * @return {boolean} - True if successful, false otherwise
  */
-export async function createApikey(data) {
-  if (!data.id) {
+export async function createApikey(id = false, data) {
```

Comment:
We have to be a lot more careful when changing function signatures like this.
Is there a need to change the signature of this function?
Because this is a breaking change for now obvious reason.

If it's needed, please explain and update the signature docs above the function.
If it's not needed, please refactor to only use data.